### PR TITLE
To-Do add and Remove functionalities

### PR DESCRIPTION
**Pull request by maaloum**
### In this PR, I :
- Created a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/addTask.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// import displayTodo from './displayTasks.js';
+import getTasks from './getTasks.js';
+
+const addTask = (task) => {
+  const todos = getTasks();
+  todos.push(task);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  // displayTodo();
+};
```
Comment (DeliceLydia) [inline src/addTask.js]: - [ ] The property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). please address that!
task

Comment (DeliceLydia) [reply src/addTask.js]: - [ ] Make sure after deleting a task you should update all remaining items' indexes, so they represent the current list order and are unique.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @maaloum, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! 
Highlights for your project:
- Awesome README
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @maaloum, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**